src/dicom_parser/series.py

```diff
@@ -203,12 +203,7 @@ def get_spatial_resolution(self) -> Tuple[float]:
         Tuple[float]
             Spatial resolution in millimeters
         """
-        sample_header = self[0].header
-        pixel_spacing = list(sample_header.get("PixelSpacing"))
-        slice_thickness = sample_header.get("SliceThickness")
-        if slice_thickness:
-            pixel_spacing.append(slice_thickness)
-        return tuple(pixel_spacing)
```

Comment:
This functionality was moved to the `Image` class, and therefore `Series` now simply retrieves the same property value from the sample image.